### PR TITLE
Update TpmDeviceTest.md

### PR DESCRIPTION
**Pull request by saraclay**

### CODE REVIEW

File: en-US/Samples/TpmDeviceTest.md [hunk 1]
```diff
@@ -7,5 +7,31 @@ permalink: /en-US/Samples/TpmDeviceTest.htm
 lang: en-US
 ---
 
+# Confusing TPM with the Azure IoT Hub
```
Comment (dcrck) [inline en-US/Samples/TpmDeviceTest.md]: I agree, the TPM is confusing

File: en-US/Samples/TpmDeviceTest.md [hunk 2]
```diff
@@ -7,5 +7,31 @@ permalink: /en-US/Samples/TpmDeviceTest.htm
...
+
+To connect to the Azure IoT Hub from a provisioned device, us the TpmDevice class from the Microsoft.Devices.Tpm library (available as
```
Comment (dcrck) [inline en-US/Samples/TpmDeviceTest.md]: Typo, should be "use the TpmDevice..."

File: en-US/Samples/TpmDeviceTest.md [hunk 3]
```diff
@@ -7,5 +7,31 @@ permalink: /en-US/Samples/TpmDeviceTest.htm
...
+the NuGet package). Get the device information stored in the desired slot (typically slow 0), then retrieve the name of the IoT Hub,
```
Comment (dcrck) [inline en-US/Samples/TpmDeviceTest.md]: should be "slot 0"

File: en-US/Samples/TpmDeviceTest.md [hunk 4]
```diff
@@ -7,5 +7,31 @@ permalink: /en-US/Samples/TpmDeviceTest.htm
...
+the device ID, and the SAS token (the string containing the HMAC produced from the shared access key) and use that to create _DeviceClient_:
```
Comment (dcrck) [inline en-US/Samples/TpmDeviceTest.md]: create the DeviceClient

File: en-US/Samples/TpmDeviceTest.md [hunk 5]
```diff
@@ -7,5 +7,31 @@ permalink: /en-US/Samples/TpmDeviceTest.htm
...
+
+```
+TpmDevice myDevice = new TpmDevice(0); // Use TPM slow 0
```
Comment (dcrck) [inline en-US/Samples/TpmDeviceTest.md]: slot 0

File: en-US/Samples/TpmDeviceTest.md [hunk 6]
```diff
@@ -7,5 +7,31 @@ permalink: /en-US/Samples/TpmDeviceTest.htm
...
+    AuthenticationMethodFactory.
+        CreateAuthenticationWithToken(deviceId, sasToken), TransportType.Amqp);
+
+var str = "Hello, Cloud!";
+var message = new Message(Encoding.SCII.GetBytes(str));
+
+await deviceClient.SendEventAsync(message);
+```
+
+At this point, you have a connected _deviceClient_ object that you can use to send and receive message. You can view the full working sample [here](https://github.com/ms-iot/samples/tree/develop/Azure/TpmDeviceTest).
```
Comment (dcrck) [inline en-US/Samples/TpmDeviceTest.md]: messages

File: en-US/Samples/TpmDeviceTest.md (hunk 1 above)
Comment (saraclay) [reply en-US/Samples/TpmDeviceTest.md]: Oh no... how did this even happen..

File: en-US/Samples/TpmDeviceTest.md (hunk 3 above)
Comment (saraclay) [reply en-US/Samples/TpmDeviceTest.md]: Goodness what was I doing...

**Review by dcrck — APPROVED** [review]

**Review by rachitb777 — APPROVED** [review]

**Pull request merged**